LICENSE

```diff
@@ -1,7 +1,5 @@
 MIT License
 
-Copyright (c) 2020 Dräger
```

Comment:
I think this change contradicts another part of the license:

> The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

since it is referring to 

> The above copyright notice

Reply:
done